src/components/BookDetail/BookDetail.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import books from '../../assets/books.json';
+
+import BookInformation from './BookInformation';
+
+class BookDetail extends React.Component {
+  state = {
+    book: books.find(book => book.id === parseInt(this.props.match.params.id, 10)) // eslint-disable-line react/prop-types
+  };
+
+  render() {
+    return (
+      <div>
```

Comment:
you don't need this wrapper div here

Reply:
This was here because I knew I was going to add more components, but I'll remove it.